docs/ManualClusterSetup.md

```diff
@@ -4,14 +4,17 @@ This document describes requirements and best practices for setting up a cluster
 
 ## Requirements
 
-2 servers (bare metal or VMs) with a modern 64-bit Linux OS, and Layer-2 network (Ethernet) connectivity between them.
+2 servers (bare metal or VMs) with a modern 64-bit Linux OS, each with a unique hostname.
 
 They must have the following software installed.
  * Docker v1.8 or greater: [Experimental Docker](https://experimental.docker.com)
  * etcd installed and available on each node: [etcd Documentation](https://coreos.com/etcd/docs/2.0.8/)
+ * Consul installed on one node [Consul Documentation](https://www.consul.io/docs/index.html)
```

Comment:
For consul, I think it's better to be consistent with how we describe our need for etcd. i.e. Consul cluster installed and accessible from each compute node.